modules/installation-configuration-parameters.adoc

```diff
@@ -1201,6 +1205,18 @@ Additional IBM Cloud VPC configuration parameters are described in the following
 |The instance type for all IBM Cloud VPC machines.
 |Valid IBM Cloud VPC instance type, such as `bx2-8x32`. [^2^]
 
+|`platform.ibmcloud.<parameter_name_for_vpc>`
+| The name of the existing VPC that you want to deploy your cluster to.
+| String.
+
+|`platform.ibmcloud.<parameter_name_for_controlplane_subnet>`
+| The name of the existing subnet in your VPC that you want to deploy your control plane machines to.
```

Comment:
The name(s) of the existing subnet(s) in your VPC that you want to deploy your control plane machines to. A subnet should be specified for each availability zone. Subnet IDs are not supported.

Reply:
Done.